app/screens/edit_channel/edit_channel.js

```diff
@@ -92,6 +92,7 @@ export default class EditChannel extends PureComponent {
         this.state = {
             error: null,
             updating: false,
+            updateChannelRequest: this.props.updateChannelRequest,
```

Comment:
we should remove this property as it is being deprecated, but I believe that is out of scope of this PR.

Reply:
Yeah, it's out of scope. I believe we're only removing the request status that isn't being used anywhere, so no changes are needed here.